tests/conftest.py

```diff
@@ -1,34 +1,74 @@
 from pathlib import Path
 import pytest, os, uuid, shutil
+from typing import List
 
-from sand.models.base import init_db
+from sand.models.base import db, init_db
+from sand.models import all_tables
+from werkzeug.datastructures import FileStorage
+from sand.controllers.helpers.upload import (
+    CSVParserOpts,
+    UploadingTable,
+    get_extension,
+    parse_upload,
+    save_upload,
+)
+import json
 
 
-@pytest.fixture()
+@pytest.fixture
 def client():
-    tempdir = Path("/tmp") / str(uuid.uuid4())
-
     try:
-        tempdir.mkdir()
-        shutil.copyfile(
-            Path(__file__).parent / "resources/sand.db", str(Path(tempdir) / "sand.db")
-        )
-        init_db(str(Path(tempdir) / "sand.db"))
+        init_db(":memory:")
+        db.create_tables(all_tables, safe=False)
 
-        from sand.app import app
         from sand.config import SETTINGS, _ROOT_DIR
+        from sand.app import app
+
+        externaldb = _ROOT_DIR / "tests/resources/data/databases"
```

Comment:
build externaldb from files instead of prebuilding them. rocksdb database may not be compatible in different OS

Reply:
Updated the fixtures